classes/log/SubmissionEmailLogEntry.php

```diff
@@ -7,131 +7,69 @@
  * Copyright (c) 2003-2021 John Willinsky
  * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
  *
- * @class SubmissionEmailLogEntry
+ * @enum SubmissionEmailLogEntry
  *
  * @ingroup log
  *
- * @see SubmissionEmailLogDAO
- *
- * @brief Base class for describing an entry in the submission email log.
+ * @brief Enumeration for submission email log event types.
  */
 
 namespace PKP\log;
+use PKP\log\core\EmailLogEventType;
 
-class SubmissionEmailLogEntry extends EmailLogEntry
+enum SubmissionEmailLogEntry: int implements EmailLogEventType
```

Comment:
I'd suggest renaming these enums to reflect their changed relationship with the `EmailLogEntry` model. This should be something like `SubmissionEmailLogEntryType`.

Reply:
updated